andela/troupon
# Ft troupon add shipping info 126489251

#### What does this PR do?

This PR allows users to add in shipping details just before purchasing a deal. 

#### Description of Task to be completed?

Users should be required to add in shipping information before purchasing a deal (that is physically delivered)

#### How should this be manually tested?
When you view cart after selecting a deal click on the proceed to checkout button and you'll be redirected to the enter shipping details page. You should then be able to see a form, you can fill in the details and proceed to pay for the deal.

#### What are the relevant pivotal tracker stories?

#126489251

#### Screenshots:
<img width="788" alt="screen shot 2016-08-03 at 12 35 08 pm" src="https://cloud.githubusercontent.com/assets/15860371/17361000/d871ca60-5976-11e6-93ff-81308710e721.png">
